spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java

```diff
@@ -86,12 +86,12 @@ public TransactionAwareConnectionFactoryProxy(ConnectionFactory targetConnection
 	/**
 	 * Delegates to {@link ConnectionFactoryUtils} for automatically participating in Spring-managed transactions.
 	 * <p>
-	 * The returned {@link ConnectionFactory} handle implements the {@link ConnectionProxy} interface, allowing to
+	 * The returned {@link ConnectionFactory} handle implements the {@link org.springframework.jdbc.datasource.ConnectionProxy} interface, allowing to
```

Comment:
Please revert this change.

This is obsolete documentation which needs to be reworked separately.